### ISSUE TITLE
New Trunk-Recorder

### ISSUE THREAD

**Issue by Dewey3** [chuot/rdio-scanner#61]
Greetings!  Just an FYI... Trunk-Recorder was upgraded a couple days ago to v4.  Part of the update includes a name change to the audio files.  Unless I am doing something wrong, Rdio-Scanner Dirwatch is not pulling the new file names.

**Comment by Dewey3** [comment]
First, thank you for all that you have done, and continue to do.  Yesterday, I actually tried pointing the Dirwatch to m4a files, but that still didn't work.  I've also noticed that I am no longer ingesting json files either, but that could very well be because of all the changes I am making trying to make it work.  I will continue to work with it today.

**Comment by JayManCanada6014** [comment]
Trunk will upload .wav files so make sure in your dir-watch settings it is set for .wav and not mp3 or m4a.  And then that your mask is setup properly.

Also why dir-watch and not using the API to upload?  Just curious.

**Comment by Dewey3** [comment]
`Trunk will upload .wav files so make sure in your dir-watch settings it is set for .wav and not mp3 or m4a`:  I tried .wav files first, and they were not not ingested.  Just for testing purposes, I also tried putting "m4a" on the "Extension" line, but that didn't work.

`And then that your mask is setup properly`:  I know that the mask will extract metadata from the file name, but I didn't think that "Mask" had an impact on actually ingesting the audio files.  Even then, Trunk-Recorder-v4 has a modified file naming convention which includes sequential numbering, which I think _may_ be part of why the audio files are not being recognized by Rdio-Scanner.

`Also why dir-watch and not using the API to upload`:  This is local setup, so after Rdio-Scanner has ingested the local audio file, it is uploaded with Rdio-Scanner's "Downstream".
